edxval/api.py

```diff
@@ -317,26 +318,46 @@ def get_url_for_profile(edx_video_id, profile):
 def _get_videos_for_filter(
         video_filter,
         sort_field=None,
-        sort_dir=SortDirection.asc
+        sort_dir=SortDirection.asc,
+        **kwargs
 ):
     """
     Returns a generator expression that contains the videos found, sorted by
     the given field and direction, with ties broken by edx_video_id to ensure a
     total order.
     """
-    videos = Video.objects.filter(**video_filter)
+    videos_qs = Video.objects.filter(**video_filter)
```

Comment:
its a query set thats why the name to keep it separate from videos below which has the list of videos